src/styles/style-guide.scss

```diff
@@ -39,6 +39,15 @@
 		left: -130px;
 		transform: scale( 1.75 );
 		top: 34px;
+
+		@media #{ $mq-md-down } {
```

Comment:
Can we do this the other way round and have it `$mq-md-up` so we don't need to reset everything in the media query?